jobscheduler/scheduler-utils/src/java/org/sakaiproject/scheduler/util/AutoProvisionJobs.java

```diff
@@ -54,13 +54,18 @@
 
     public void init() throws ParserConfigurationException, XPathException, ParseException, IOException, ValidationException, SchedulerException, SAXException, ClassNotFoundException {
 
+        Scheduler scheduler = schedulerManager.getScheduler();
```

Comment:
Since the SchedulerManager Bean is injected via spring it should never be non null. If schedulerManager.getScheduler() is returning null then there is likely some timing issue going on.

Reply:
The schedulerManager isn't null, but getScheduler() was returning null because it wasn't set up after this in init.

I can move this abort down further but it really seems like everything in this init shouldn't be running if startScheduler=false. It's just doing too much. I moved it down right before the jobs are processed and removed the null checks and it's loading up and seems to still be working as expected.